pages/careers/index.js

```diff
@@ -3,8 +3,20 @@ import Mission from './mission.md'
 import Intro from './intro.md'
 import Values from './values.md'
 import { Jobs } from '@components/jobs'
-
-import { fetchStats } from '@common/lib'
+import fetch from 'cross-fetch'
+const fetchStats = async () => {
+  try {
+    const res = await fetch(
+      'https://blockstack-site-api.herokuapp.com/v1/stats'
```

Comment:
This should be an issue in the API repo probably, but this returns 0 for `slack_users` and `forum_users`.

Reply:
@yknl do you know which repo contains the code for the blockstack site api? We can probably just integrate most of the calls into this repo now.

Reply:
Because the Slack API call uses an API token, we should keep it on the API server.

Reply:
I made an issue at https://github.com/blockstack/blockstack.org-api/issues/3